spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/sql/AbstractJdbcDatasourcePropertiesUpdater.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.cloud.autoconfigure.sql;
+
+import com.microsoft.azure.spring.cloud.context.core.AzureAdmin;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.util.Assert;
+import org.springframework.util.StringUtils;
+
+/**
+ * Base class of {@link JdbcDataSourcePropertiesUpdater}
+ *
+ * @author Warren Zhu
+ */
+public abstract class AbstractJdbcDatasourcePropertiesUpdater implements JdbcDataSourcePropertiesUpdater {
+    protected static final Log LOGGER = LogFactory.getLog(SqlServerJdbcDataSourcePropertiesUpdater.class);
```

Comment:
Why the logger name `SqlServerJdbcDataSourcePropertiesUpdater` is different with the class name `AbstractJdbcDatasourcePropertiesUpdater`?

Reply:
Typo. Should be later one.